ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -29,3 +29,32 @@ def test_numpy_iftt(
         axis=-1,
         norm=None
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.ifft2",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
just one quick side note, is it only supporting `float` or can it take numbers in general? if yes, please test on `numeric` instead, thanks!

Reply:
right, it can take other data types as well. Will do now.